protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt

```diff
@@ -0,0 +1,8 @@
+#Wed Dec 17 17:11:53 CET 2014
+#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd
```

Comment:
#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd
0x12345678_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=A1##02##01##Bla bla-bla friendly bla-bla name ;-) ##descrip
0x00123456_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=00##22##44##frnm##no description

The  3 lines above are just empty examples.

The 4 lines below are examples we referred to in org.eclipse.smarthome.protocols.enocean.sample.client (i.e. references are on 0x018c0874, etc.).

0x018c0874_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=F6##99##99##Eltako Smoke Detector##(F6)
0x01819912_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=F6##05##01##AfrisoLab Water Sensor##Liquid Leakage Sensor (mechanic energy harvester) (F6-05-01)
#0x11111111_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=D2##01##08##D2-01-08##Electronic switches and dimmers with Energy Measurement and Local Control
0x00851656_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=D2##A0##01##AfrisoLab Water Valve##Standard Control (BI-DIR) (D2-A0-01)

Is it ok for ESH?

Antonin.


De : Kai Kreuzer [mailto:notifications@github.com]
Envoyé : jeudi 5 mars 2015 17:12
À : eclipse/smarthome
Cc : CHAZALET Antonin OAB
Objet : Re: [smarthome] EnOcean OSGi Base Driver (https://bugs.eclipse.org/bugs/show_bug.cgi?id=457338) (#155)


In protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt<https://github.com/eclipse/smarthome/pull/155#discussion_r25875521>:

> @@ -0,0 +1,8 @@

> +#Wed Dec 17 17:11:53 CET 2014

> +#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd

Well, I was also wondering about all other lines in this file as it refers to specific products from specific manufacturers. Why is this needed inside a generic base driver? If it is only for tests, it should at least not end up in the final jar.

—
Reply to this email directly or view it on GitHub<https://github.com/eclipse/smarthome/pull/155/files#r25875521>.

_________________________________________________________________________________________________________________________

Ce message et ses pieces jointes peuvent contenir des informations confidentielles ou privilegiees et ne doivent donc
pas etre diffuses, exploites ou copies sans autorisation. Si vous avez recu ce message par erreur, veuillez le signaler
a l'expediteur et le detruire ainsi que les pieces jointes. Les messages electroniques etant susceptibles d'alteration,
Orange decline toute responsabilite si ce message a ete altere, deforme ou falsifie. Merci.

This message and its attachments may contain confidential or privileged information that may be protected by law;
they should not be distributed, used or copied without authorisation.
If you have received this email in error, please notify the sender and delete this message and its attachments.
As emails may be altered, Orange is not liable for messages that have been modified, changed or falsified.
Thank you.